code/modules/projectiles/projectile/beams.dm

```diff
@@ -144,6 +144,34 @@
 /obj/projectile/beam/emitter/singularity_pull()
 	return //don't want the emitters to miss
 
+//This is basically a cannon beam
+/obj/projectile/beam/emitter/heavy
+	name = "heavy emitter beam"
+	icon_state = "emitter_heavy"
+	damage = 600
+	wound_bonus = 70
+	armour_penetration = 100
+
+/obj/projectile/beam/emitter/heavy/on_hit(atom/target, blocked)
+	. = ..()
+
+	if(iscarbon(target))
+		var/mob/living/carbon/carbon_target = target
+		for(var/i in 1 to 3)
+			var/obj/item/bodypart/bodypart = pick(carbon_target.bodyparts)
+			var/type_found = pick(GLOB.global_wound_types[WOUND_BURN])
+			var/datum/wound/burnies = new type_found()
+			burnies.apply_wound(bodypart)
+
+	if(istype(target,/obj/machinery/power/supermatter_crystal))
+		return
+
+	explosion(src,rand(0,1),1+rand(0,1),2+rand(0,1))
+
+/obj/projectile/beam/emitter/heavy/on_range()
+	explosion(src,rand(1,2),1+rand(1,2),2+rand(1,2))
```

Comment:
why is this larger than the on-hit explosion?

Reply:
wanted to make the projectile reaching it's limit have a bigger umph